demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -1867,3 +1870,43 @@ def test_IsContentItemNameContainTrailingSpacesValidator_fix(
         results.message
         == f"Removed trailing spaces from the {', '.join(fields_with_trailing_spaces)} fields of following content items: {VALUE_WITH_TRAILING_SPACE.rstrip()}"
     )
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_msg",
+    [
+        pytest.param(
+            [
+                create_integration_object(name="invalid_integration_name"),
+            ],
+            "The folder name 'invalid_integration_name' should be without any separator.",
+            id="an invalid integration name",
+        ),
+        pytest.param(
+            [
+                create_integration_object(name="invalidIntegrationName"),
+            ],
+            "",
+            id="a valid integration name.",
+        ),
+        pytest.param(
+            [
+                create_script_object(name="invalid-script-name"),
+            ],
+            "The folder name 'invalid-script-name' should be without any separator.",
+            id="an invalid script name",
+        ),
+        pytest.param(
+            [
+                create_script_object(name="invalidScriptName"),
+            ],
+            "",
+            id="a valid script name",
+        ),
+    ],
+)
+def test_IsFolderNameHasSeparatorsValidator_is_valid(content_items, expected_msg):
+    result = IsFolderNameHasSeparatorsValidator().is_valid(content_items)
+
+    if result:
```

Comment:
```suggestion
    result = IsFolderNameHasSeparatorsValidator().is_valid(content_items)

    if result:
```
Better to test the number of results here as well
If results = [] in the first case then we'll still pass the test.

Reply:
I added,
I hope that's what you meant